scorecardIntegration.test.js

```diff
@@ -0,0 +1,112 @@
+const Scorecard = require('./scorecard');
+
```

Comment:
Large range of tests covering the various possibilities of scores, including edge cases.

Reply:
Should there be unit tests for scorecard and scorecalculator?